share/spack/gitlab/cloud_pipelines/stacks/hep/spack.yaml

```
@@ -65,12 +74,28 @@ spack:
   - py-vector
   - pythia8 +evtgen +fastjet +hdf5 +hepmc +hepmc3 +lhapdf ~madgraph5amc +python +rivet ~root # pythia8 and root circularly depend
   - rivet hepmc=3
-  - root +davix +dcache +examples +fftw +fits +fortran +gdml +graphviz +gsl +http +math +minuit +mlp +mysql +opengl +postgres +pythia8 +python +r +roofit +root7 +rpath ~shadow +spectrum +sqlite +ssl +tbb +threads +tmva +unuran +vc +vdt +veccore +webgui +x +xml +xrootd
+  - root ~cuda
   - sherpa +analysis ~blackhat +gzip +hepmc3 +hepmc3root +lhapdf +lhole +openloops +pythia ~python ~recola ~rivet +root +ufo
   - thepeg ~rivet
-  - vecgeom +gdml +geant4 +root
+  - vecgeom ~cuda
   - xrootd +davix +http +krb5 +python +readline +scitokens-cpp
 
+  # CUDA NOARCH
+  - root +cuda +cudnn +tmva-gpu
+
+  # CUDA 80
+  #- acts +cuda +traccc cuda_arch=80
+  - celeritas +cuda +vecgeom cuda_arch=80
+  - vecgeom +cuda cuda_arch=80
+
+  # CUDA 90
+  #- acts +cuda +traccc cuda_arch=90
+  - celeritas +cuda +vecgeom cuda_arch=90
+  - vecgeom +cuda cuda_arch=90
```

It should be possible to compile multiple cuda architectures simultaneously right?

---

Not sure if `cuda_arch=80,90` works, or do you mean something else?

---

Yes, it looks like Spack with CMake's `cuda_arch` variant supports multiple simultaneous values: so you should be able to use
```suggestion
  - vecgeom +cuda cuda_arch=80,90
```
to build in multiple runtimes.

---

I'm just going to change it to a single `cuda_arch=80`. Not sure there's value in the multiple variants in CI.